primitives/consensus/beefy/Cargo.toml

```diff
@@ -24,10 +24,14 @@ sp-runtime = { version = "24.0.0", default-features = false, path = "../../runti
 sp-std = { version = "8.0.0", default-features = false, path = "../../std" }
 strum = { version = "0.24.1", features = ["derive"], default-features = false }
 lazy_static = "1.4.0"
+w3f-bls = { version = "0.1.3", default-features = false, optional = true}
 
 [dev-dependencies]
 array-bytes = "4.1"
 sp-keystore = { version = "0.27.0", path = "../../keystore" }
+w3f-bls = { version = "0.1.3", features = ["std"]} 
+hex = { version = "0.4", default-features = false}
+hex-literal = "0.3.4"
```

Comment:
please use `array_bytes` crate for the same goal (we're trying to reduce dependencies offering duplicated functionality)

Reply:
This applies to `sp_core::bls` as well

Reply:
Removed and replaced in `sp_core::bls` as well.